dask/dataframe/dispatch.py

```diff
@@ -22,6 +22,12 @@
 tolist_dispatch = Dispatch("tolist")
 is_categorical_dtype_dispatch = Dispatch("is_categorical_dtype")
 union_categoricals_dispatch = Dispatch("union_categoricals")
+percentile_dispatch = Dispatch("percentile_dispatch")
+
+
+def _percentile(a, q, interpolation="linear"):
+    func = percentile_dispatch.dispatch(type(a))
+    return func(a, q, interpolation)
```

Comment:
I don't think this is needed as `Dispatch` objects have a `__call__` method with this same logic 

https://github.com/dask/dask/blob/bb0a2d2f56b0e12ce0fa23a8aaad1203f8da36f7/dask/utils.py#L570-L575

Reply:
will handle this too in a followup PR.

Reply:
PR link: https://github.com/dask/dask/pull/8055